Decisions.TruCap/Steps/DocumentMonitorSteps.cs

```diff
@@ -50,35 +52,39 @@ public class DocumentMonitorSteps
             }
             
             string baseUrl = ModuleSettingsAccessor<TruCapSettings>.GetSettings().GetBaseDocumentMonitorUrl(overrideBaseUrl);
-            Task<HttpResponseMessage> response = TruCapRest.TruCapGet($"{baseUrl}/DocumentId/{documentId}", authentication);
+            HttpResponseMessage response = TruCapRest.TruCapGet($"{baseUrl}/DocumentId/{documentId}", authentication);
 
             try
             {
-                return DocumentMonitorResponse.JsonDeserialize(await response.Result.Content.ReadAsStringAsync());
+                Task<string> resultTask = response.Content.ReadAsStringAsync();
+                resultTask.Wait();
+                return DocumentMonitorResponse.JsonDeserialize(resultTask.Result);
             }
             catch (Exception ex)
             {
-                throw new Exception(await response.Result.Content.ReadAsStringAsync());
+                throw new Exception(response.Content.ReadAsStringAsync().Result);
```

Comment:
Same comment as above. Since these are steps we want readable exception messages.